cmk/base/plugins/agent_based/prism_host_networks.py

```diff
@@ -0,0 +1,118 @@
+    data = load_json2(string_table)
+    for element in data:
+        parsed.setdefault(element.get("name", "unknown"), element)
+    return parsed
+
+
+register.agent_section(
+    name="prism_host_networks",
+    parse_function=parse_prism_host_networks,
+)
+
+
+def _create_interface(section: Section) -> interfaces.Section[interfaces.InterfaceWithRates]:
+    return [
+        interfaces.InterfaceWithRates(
+            attributes=interfaces.Attributes(
+                index=str(index),
+                descr=name,
+                alias=name,
+                type="6",
+                speed=(
+                    0
+                    if not raw_stats["link_speed_in_kbps"]
+                    else float(raw_stats["link_speed_in_kbps"]) * 1000
+                ),
+                oper_status=("1" if raw_stats["link_speed_in_kbps"] else "2"),
+                phys_address=interfaces.mac_address_from_hexstring(
+                    raw_stats.get("mac_address", "")
+                ),
+            ),
+            rates=interfaces.Rates(
+                in_octets=float(raw_stats["stats"]["network.received_bytes"]) / 30,
+                in_ucast=float(raw_stats["stats"]["network.received_pkts"]) / 30,
+                in_mcast=float(raw_stats["stats"]["network.multicast_received_pkts"]) / 30,
+                in_bcast=float(raw_stats["stats"]["network.broadcast_received_pkts"]) / 30,
+                in_disc=float(raw_stats["stats"]["network.dropped_received_pkts"]) / 30,
+                in_err=float(raw_stats["stats"]["network.error_received_pkts"]) / 30,
+                out_octets=float(raw_stats["stats"]["network.transmitted_bytes"]) / 30,
+                out_ucast=float(raw_stats["stats"]["network.transmitted_pkts"]) / 30,
+                out_mcast=float(raw_stats["stats"]["network.multicast_transmitted_pkts"]) / 30,
+                out_bcast=float(raw_stats["stats"]["network.broadcast_transmitted_pkts"]) / 30,
+                out_disc=float(raw_stats["stats"]["network.dropped_transmitted_pkts"]) / 30,
+                out_err=float(raw_stats["stats"]["network.error_transmitted_pkts"]) / 30,
+            ),
+            get_rate_errors=[],
+        )
+        for index, (name, raw_stats) in enumerate(sorted(section.items()))
+    ]
+
+
+def discovery_prism_host_networks(
+    params: Sequence[Mapping[str, Any]], section: Section
+) -> DiscoveryResult:
+    yield from interfaces.discover_interfaces(
+        params,
+        _create_interface(section),
+    )
+
+
+def _check_prism_host_network(
+    item: str, params: Mapping[str, Any], section: Section, value_store: MutableMapping[str, Any]
+) -> CheckResult:
+    yield from interfaces.check_multiple_interfaces(
+        item,
+        params,
+        _create_interface(section),
+        value_store=value_store,
+    )
+
+
+def check_prism_host_networks(
+    item: str, params: Mapping[str, Any], section: Section
+) -> CheckResult:
+    data = section.get(item)
```

Comment:
are you sure that those statements are really needed?
i would think that this case is already handled by `interfaces.check_multiple_interfaces`.